mobile/ios/ios-core.ts

```diff
@@ -543,7 +545,11 @@ export class MobileDevice implements Mobile.IMobileDevice {
 	}
 
 	public deviceMountImage(devicePointer: NodeBuffer, imagePath: NodeBuffer, options: NodeBuffer, mountCallBack: NodeBuffer): number {
-		return this.mobileDeviceLibrary.AMDeviceMountImage(devicePointer, imagePath, options, mountCallBack, null);
+		if(hostInfo.isDarwin()) {
+			return this.mobileDeviceLibrary.AMDeviceMountImage(devicePointer, imagePath, options, mountCallBack, null);
+		}
+
+		this.$errors.fail("AMDeviceMountImage is exported only on Darwin OS");
```

Comment:
I believe that this error message is baffling to the user. Consider giving a more helpful user message.